cmd.go

```diff
@@ -191,8 +192,15 @@ func (ctx *Context) Setenv(key, value string) error {
 }
 
 // AbsPath returns an absolute representation of path, with relative paths
-// interpreted as relative to ctx.Dir.
+// interpreted as relative to ctx.Dir and with "~/" replaced with users
+// home dir.
 func (ctx *Context) AbsPath(path string) string {
+	if len(path) >= 2 && path[:2] == "~/" {
```

Comment:
You can use "strings.HasPrefix"
If we're doing this, what about ~user/ and what about Windows, etc?
Typically ~ is expanded by the shell rather than the commands themselves.

Was this for a specific bug. Because it does mean that you can't quote args to not treat ~ as $HOME, because bash will take the quote and then pass it to us and we'll still expand it.
Probably still ok, but it would be good to understand why ~ expansion wasn't working in someone's shell.